src/write_gcm.c

```diff
@@ -4183,6 +4193,10 @@ static int wg_write(const data_set_t *ds, const value_list_t *vl,
           metadata);
     wg_payload_destroy(to_remove);
   }
+  if (drop_count != 0) {
```

Comment:
Ah I see what you're saying. Yes it is the latter. When the next minute worth of metrics are going to be dispatched at this point we don't know if they will remain in the queue or they will be dropped. Fixed.